src/components/EventListCard/Modal/EventListCardModals.tsx

```
@@ -131,17 +134,75 @@ function EventListCardModals({
+        data = result.data;
+      } else {
+        // Recurring instance - handle based on selected option
+        switch (deleteOption) {
+          case 'single':
+            const singleResult = await deleteSingleInstance({
+              variables: {
+                input: {
+                  id: eventListCardProps._id,
+                },
+              },
+            });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)